templates/footer.twig

```diff
@@ -1,4 +1,9 @@
 <footer id="footer" class="site-footer site-footer--{{ nav_type }}">
+	{% if new_design_country_selector and nav_type != 'minimal' %}
+	<div id="country-selector" class="site-footer__country-selector">
```

Comment:
Could you change `site-footer__country-selector` to `footer-country-selector` or sth like that? since `site-footer__country-selector` is referring to a BEM naming fomat but we're not using that. I initially used BEM on the new Footer design but I had to changed it and respect the style guide of class names we have.